samples/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(NovelRTTest VERSION 0.0.1)
```

Comment:
Why `Test` rather than `NovelRTSamples`?

Reply:
Because that is the current name of the window that is produced, and it it was 3 AM when I wrote this 🤣 I'll modify it.